clients/shisui/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG branch=latest
+
+FROM ghcr.io/optimism-java/shisui:$branch
```

Comment:
```suggestion
ARG baseimage=ghcr.io/optimism-java/shisui
ARG tag=latest

FROM $baseimage:$tag
```

Reply:
Could you make this change as well? As this format is used to pass which tag you want to use when running hive tests

Reply:
Got it